code/game/objects/items/control_wand.dm

```diff
@@ -32,12 +32,16 @@
 	switch(mode)
 		if(WAND_OPEN)
 			mode = WAND_BOLT
+			desc = "Remotely controls airlocks. Current mode: toggle bolts"
 		if(WAND_BOLT)
 			mode = WAND_EMERGENCY
+			desc = "Remotely controls airlocks. Current mode: toggle emergancy access"
```

Comment:
*cough
```suggestion
			desc = "Remotely controls airlocks. Current mode: toggle emergency access"
```

Reply:
Ironic that i made a spelling mistake fix PR after this.